File: poo/JS/car.js
var result = document.getElementById("result");
class Car {
    constructor(license, passengers, driver) {
        this.id;
        this.license = license;
        this.driver = driver;
        this.passengers = passengers;
    }
    printDataCar() {
        result.innerHTML = "Nombre del Conductor: " + this.driver.name + "<br />Documento del Conductor: " + this.driver.document + "<br />Pasajeros: " + this.passengers + "<br />Licencia: " + this.license
        console.log(this.driver.name);
        console.log(this.driver.document); 
        console.log(this.passengers);
        console.log(this.license);       
    }
}

File: poo/Python/car.py
from account import Account

class Car:
    id         = int
    license    = str
    driver     = Account("","")
    passengers = int

    def __init__(self, license, passengers, driver) -> None:
        self.license = license
        self.driver = driver
        self.passengers = passengers

    def printDataCar(elem):
        elements = (vars(elem))
        driver = (vars(elem.driver))
        print(elements)
        print(driver)